guides/release/testing/testing-components.md

```diff
@@ -103,31 +121,33 @@ module('Integration | Component | pretty-color', function(hooks) {
     // set the outer context to red
     this.set('colorValue', 'red');
 
-    await render(hbs`<PrettyColor @name={{this.colorValue}} />`);
+    await render(<template> <PrettyColor @name={{this.colorValue}} /> </template>);
 
     assert.equal(this.element.querySelector('div').getAttribute('style'), 'color: red', 'starts as red');
 
     this.set('colorValue', 'blue');
 
-    assert.equal(this.element.querySelector('div').getAttribute('style'), 'color: blue', 'updates to blue');  });
+    assert.equal(this.element.querySelector('div').getAttribute('style'), 'color: blue', 'updates to blue');  
```

Comment:
```suggestion
    assert.strictEqual(this.element.querySelector('div').getAttribute('style'), 'color: blue', 'updates to blue');  
```

Reply:
I'm trying to keep the changes as minimal as possible here 🤔 is there value in changing it to strictEqual here?

Reply:
`assert.equal` will cause a lint error

Reply:
I generated a brand new Ember app using `--strict` and it doesn't have a lint error with this exact code 🤔 maybe it's something that we used to have?

Reply:
there is a lint error telling us not to use `@ember/runloop` 🙈 but that's a different question for a different day

Reply:
Ah, it was, and it is only fixed in 6.8 https://github.com/ember-cli/ember-cli/pull/10707